### ISSUE TITLE
Setup VSCode to run webpack on file save

### ISSUE THREAD

**Issue by gannonprudhomme** [aggie-coding-club/Automatic-Aggie-Scheduler#16]
Should find out how to configure VSCode to run webpack automatically anytime there's a change to a `.tsx` file.

**Comment by firejake308** [comment]
Hint for whoever wants to work on this: I had this working for my last Vue project, and it worked by using `webpack-dev-server`, so you may want to try that out

**Comment by gannonprudhomme** [comment]
Also my implementation: 
While it doesn't run on save, both vim & non-vim users could add:

To `.vscode/tasks.json`:

```
{
    "version": "2.0.0",
    "tasks": [
        {
            "type": "npm",
            "script": "dev",
            "path": "autoscheduler/frontend/src/",
            "group": {
                "kind": "build",
                "isDefault": true
            }
        }
    ]
}
```

And to the global `keybindings.json`(do Ctrl+Shift+P then `Open Keyboard Shortcuts(JSON)`, add:

```     { "key": "ctrl+b", "command": "workbench.action.tasks.build"} ```

Which would run `npm run dev` when `Ctrl+b` is pressed.

While I did see some vim-specific keybindings/macros, I couldn't find anything that would run a command on `:w`. Also didn't look too deeply into running it on just a general save file, so not sure if that functionality exists.

**Comment by firejake308** [comment]
Closed by #69
